tfe/resource_tfe_workspace_test.go

```diff
@@ -487,7 +515,7 @@ func testAccCheckTFEWorkspaceAttributesUpdated(
 			return fmt.Errorf("Bad auto apply: %t", workspace.AutoApply)
 		}
 
-		if workspace.Operations != false {
+		if workspace.Operations != true {
```

Comment:
Why did you change this? It seems to be unrelated to your changes right?

Reply:
Same as above, this change make test pass and it's unrelated to my changes. Tests are not passing from master branch.